internal/controller/atlas/provider.go

```diff
@@ -8,16 +8,18 @@ import (
 	"runtime"
 	"strings"
 
-	"go.mongodb.org/atlas-sdk/v20231115008/admin"
+	"github.com/mongodb-forks/digest"
+	adminv20231115008 "go.mongodb.org/atlas-sdk/v20231115008/admin"
```

Comment:
nit

do we need to distinguish up to the hour, I think year moth is probably enough right?

Reply:
it's has nothing with hour, sdk versioning has an incremental 3 digits composing the major version

Reply:
true, my bad, still I do not think we need anything beyond yyyymm internally

Reply:
I'd leave as is and not deviate from the SDK pkg names

Reply:
adding to Serge's comment, I do see the possibility of having adminv20231115007 and adminv20231115008 at the same time, for example. Better to match pkg name pattern and be on the safe side of integration/compatibility.

Reply:
ok, then agreed